dc.sh

```diff
@@ -6,4 +6,4 @@ if [[ -f .secrets.* ]]; then
   source .secrets.*
 fi
 set +a
-docker-compose $@
+docker compose $@
```

Comment:
This is a newer syntax. It should work on your laptop too (but may depend on what version of docker you have installed)